app/views/users/show.html.erb

```diff
@@ -1,2 +1,29 @@
-<h1>Here is a user</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<div class="user_container" >
+  <div class="user_photo_container" >
+    <img src='<%= @user.photo %>' a='user profile picture'>
+  </div>
+  <div class="user_profile_container" >
+    <p class="user_name" ><%= @user.name %></p>
+    <p class="user_post_counter">Number of posts: <%= @user.posts_counter %></p>
+  </div>
+</div>
+<div class="user_bio_container">
+  <p class="bio">Bio</p>
+  <p class="user_bio"><%= @user.bio %></p>
+</div>
+<div class='add_post_box'>
+  <%= link_to 'Add Post', user_new_post_path(@user), class: 'add_post_btn' %>
+</div>
+<% @posts.each do |post| %>
```

Comment:
- [ ]  I get  this error when I first try to render your project:
![Screenshot from 2021-11-06 19-37-59](https://user-images.githubusercontent.com/62705947/140620369-f1fec5c1-34ff-4624-8caf-c76bf452ba6f.png)